prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py

```diff
@@ -132,17 +133,28 @@ def omen_replicate_from_tx(
         # Do as the last steps, because it calls OpenAI (costly & slow).
         if is_invalid(market.question):
             logger.info(
-                f"Skipping `{market.question}` because it seems to be an invalid question."
+                f"Skipping `{market.question}` was marked as invalid. Trying to rephrase and make it valid."
             )
-            continue
+            new_question = rephrase(market.question)
+            logger.info(f"Rephrased `{market.question}` to `{new_question}`.")
+
+            # Reinsert at the beginning of the list to reprocess with the new question
```

Comment:
What do you mean by reinserting to beginning of list?

Reply:
This was an old version - no more reordering happening now.